WcaOnRails/app/views/posts/_post.html.erb

```diff
@@ -22,6 +22,6 @@
     <div>
       <em>Announced by <%= post.author.name %> on <%= local_time(post.created_at) %></em>
     </div>
-    <%= post.body.html_safe %>
```

Comment:
Spacing nit: the convention I believe I've been following is to put a space after the equals sign here. (full disclosure: I'm super new to Rails, and am not married to any such decisions, but do want to be consistent)